.travis.yml

```diff
@@ -33,16 +33,21 @@ script:
 
 jobs:
   include:
-    # Define the commitlint stage
-    - stage: commitlint
+    # Define the `lint` stage (runs `yamlllint` and `commitlint`)
```

Comment:
Too many 'L's in `yamllint`? :smile:

Reply:
Oh dear, we need a comment linter as well!

Reply:
Fixed, thanks.